contrib/Shopify/sarama/option.go

```diff
@@ -51,6 +53,20 @@ func WithServiceName(name string) Option {
 	}
 }
 
+// WithDataStreams enables the Data Streams monitoring product features: https://www.datadoghq.com/product/data-streams-monitoring/
+func WithDataStreams() Option {
+	return func(cfg *config) {
+		cfg.dataStreamsEnabled = true
+	}
+}
+
+// WithGroupID tags the produced data streams metrics with the given groupID
```

Comment:
What exactly is a group ID? Will sarama users know what this is?

Reply:
A group ID is the same thing as a consumer group. I can explain that here.